ios/flutter_segment.podspec

```diff
@@ -3,21 +3,23 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'flutter_segment'
-  s.version          = '0.0.1'
-  s.summary          = 'A new flutter plugin project.'
+  s.version          = '3.1.1'
```

Comment:
Whats this number `3.1.1`? why that one?

Reply:
This is version of `flutter_segment` itself. This change is something I highlighted on my screenshot in the PR description. It makes no sense to read error about `flutter-segment` being `0.0.1`, while in reality it isn't `0.0.1`. 

Take a look at Appsflyer, for example, on how pod should look like: https://github.com/AppsFlyerSDK/appsflyer-flutter-plugin/blob/master/ios/appsflyer_sdk.podspec

Reply:
Oh got it, but the of this version is `3.11.0`.

Im not sure if we should update that version on the podspec, because then we will have to update that on every deploy too 🤔 

Maybe we could find a way to read it from the pubspec from there and avoid having to put it manually but I think that is out of the scope of this PR, WDYT?

Reply:
Made a typo in version, good catch. Will fix it.

I think it is worth updating, so errors make sense to the person during debugging.

For example, check out the error description on the screenshot for AppsFlyer. Thanks to correct version provided in their podspec I was able to understand that the issue is in different versions.

As to how to read it from the `pubspec` - yes, agree that it is a scope of a different PR. Also, not sure how to organise it correctly at the moment.

Reply:
Got it 👍🏻 

I created an issue to not lose the idea: https://github.com/la-haus/flutter-segment/issues/57